lib/metrics/AmqpMetricReporter.ts

```diff
@@ -363,15 +424,12 @@ export class AmqpMetricReporter extends ScheduledMetricReporter<AmqpMetricReport
    * @returns {Promise<void>}
    * @memberof AmqpMetricReporter
    */
-  protected handleResults(
-    ctx: OverallReportContext,
-    registry: MetricRegistry,
-    date: Date,
-    type: MetricType,
-    results: Array<ReportingResult<any, {}>>): Promise<void> {
-    const message = new Amqp.Message(JSON.stringify(results));
-
-    this.exchange.send(message);
+  protected handleResults(ctx: OverallReportContext, registry: MetricRegistry, date: Date, type: MetricType, results: Array<ReportingResult<any, Amqp.Message>>): Promise<void> {
+    results.forEach((result) => {
```

Comment:
May be more convenient
```suggestion
    results
        .filter((result) => result.result)
        .forEach((result) => this.exchange.send(result.result));
```